src/main/java/no/ndla/taxonomy/rest/v1/Resources.java

```diff
@@ -46,7 +53,7 @@ protected String getLocation() {
         return "/v1/resources";
     }
 
-    @GetMapping("/v1/resources")
+    @GetMapping("")
```

Comment:
Må du ha innhold i annotasjonen? Kan du ikkje berre ha @GetMapping eller @GetMapping() for å brukes samme som controlleren?

Reply:
Det ser ut til å funke uten :+1: